security/src/main/java/io/micronaut/security/token/propagation/HttpHeaderTokenPropagatorConfigurationProperties.java

```diff
@@ -42,7 +42,7 @@
     private static final String DEFAULT_PREFIX = HttpHeaderValues.AUTHORIZATION_PREFIX_BEARER;
     private static final String DEFAULT_HEADER_NAME = HttpHeaders.AUTHORIZATION;
 
-    private String prefix = DEFAULT_PREFIX;
+    private String headerPrefix = DEFAULT_PREFIX;
```

Comment:
why do we need this? Is not better to have it named as the getters and setters?

Reply:
It "cognitively" conflicts with the constant `PREFIX`. Since the constant is public and can't be renamed, I renamed the field.

Reply:
I see, do you think is worth renaming the constant in the next mayor version and rename the field back to `prefix`?

Reply:
I honestly don't think it's worth the effort.